packages/synapse-react-client/src/style/abstracts/_mixins.scss

```diff
@@ -185,3 +186,19 @@ $link-underline-color-default: adjust-color(
     margin-top: 16px;
   }
 }
+
+@mixin hv-center() {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin side-padding($padding: 25px) {
+  padding-left: $padding;
+  padding-right: $padding;
+}
+
+@mixin space-between() {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
These new mixins already existed as placeholders (`%hv-center` syntax), but sharing placeholders doesn't seem to be possible with the module system. I don't fully understand why/how this changed, but moving them here to be shared mixins was very easy and solved the issue.